stubs/toml/toml/encoder.pyi

```diff
@@ -0,0 +1,30 @@
+from _typeshed import SupportsWrite
+from typing import Any, Callable, Generic, Iterable, Mapping, MutableMapping, TypeVar
+
+_T = TypeVar("_T")
+
+def dump(o: Mapping[str, _T], f: SupportsWrite[str], encoder: TomlEncoder[_T] | None = ...) -> str: ...
+def dumps(o: Mapping[str, _T], encoder: TomlEncoder[_T] | None = ...) -> str: ...
+
+class TomlEncoder(Generic[_T]):
+    _dict: type[Mapping[str, _T]]
```

Comment:
This doesn't make sense to me. An encoder can encode values of any type, not only of some specific value type `_T`. But being generic over the mapping type would again be useful in `dump_sections()`, where it includes an instance of `self._dict` in the returned tuple.